scenario_player/main.py

```diff
@@ -466,10 +467,13 @@ def reclaim_eth(
     configure_logging_for_subcommand(construct_log_file_name("reclaim-eth", data_path))
 
     reclamation_candidates = get_reclamation_candidates(data_path, min_age)
+    address_to_candidate: Dict[Address, ReclamationCandidate] = {
+        to_canonical_address(c.address): c for c in reclamation_candidates
```

Comment:
I couldn't use a set of `ReclamationCandidate` because the class is not hashable, and I didn't want to go through the trouble of removing the dictionary instance now.